e2eTests/.fructose/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+const webpack = require("webpack");
+const path = require("path");
+
+module.exports = {
+  entry: {
+    app: path.join(__dirname, "./index.web.js")
+  },
+  resolve: {
+    alias: {
+      "react-native": "react-native-web"
+    },
+    extensions: [".web.js", ".js", ".ios.js", ".android.js"],
+    mainFields: ["module", "main"],
```

Comment:
Unless this is a config I've never seen before, I think plugins are supposed to be on the same level as `resolve`, so there would be a `}` here.

Reply:
It is supposed to be like this.

Reply:
How come it's different? Can't find anything on Google.